datapackage_pipelines_mojp/bagnowka/processors/download.py

```diff
@@ -0,0 +1,53 @@
+from datapackage_pipelines_mojp.common.processors.base_processors import BaseProcessor
+import json
+import logging
+import time
+
+
+class Processor(BaseProcessor):
+
+    def __init__(self, *args, **kwargs):
+        super(Processor, self).__init__(*args, **kwargs)
+
+    @classmethod
+    def _get_schema(cls):
+        # TODO change scheme
+        return {"fields": [{"name": "source", "type": "string"}, {"name": "collection", "type": "string"},
+            {"name": "name", "type": "string"},
+            {"name": "main_image_url_aws_s3", "type": "string"},
+            {"name": "description", "type": "string"},
+            {"name": "id", "type": "integer"},
+            {"name": "date_taken", "type": "string"},
+            {"name": "parsed_doc", "type": "object"}]}
```

Comment:
* source and collection fields are not necessary
* main_image_url is enough, no need to specify that it's in aws_s3 in the field name
  * you can provide this detail in the description:
    * `{"name": "main_image_url", "type": "string", "description": "stored in aws S3"}`
* date_taken should be one of the date or year field types - https://pre-v1.frictionlessdata.io/json-table-schema/#date
* parsed_doc shouldn't be needed here, you should be able to have all values available as distinct fields in the output rows